test/test_helper.rb

```diff
@@ -1,13 +1,22 @@
 ENV['RAILS_ENV'] ||= 'test'
 require_relative "../config/environment"
 require "rails/test_help"
+require "database_cleaner-active_record"
+DatabaseCleaner[:active_record].strategy = :transaction
+
 
 class ActiveSupport::TestCase
   # Run tests in parallel with specified workers
   parallelize(workers: :number_of_processors)
   include FactoryBot::Syntax::Methods
 
   # Setup all fixtures in test/fixtures/*.yml for all tests in alphabetical order.
+  def setup
```

Comment:
I was noticing bugs once I changed some things, and I solved it with these but who knows.

Reply:
Hadn't noticed on my machine! One more reason to upvote [this issue](https://github.com/piacsek/q_and_a/issues/6) 😉